### PR TITLE
Add error code 0711

### PR DESCRIPTION
**Pull request by domix**
this new error code will replace the usage of the [wrong error code](https://bitsomx.atlassian.net/browse/CORE-2154)

### CODE REVIEW

File: source/index.html.md [hunk 1]
```diff
@@ -218,6 +218,7 @@ error categories, the last two digits define specific errors.
 * 0707: 2FA Locked
 * 0708: Max attempts reached to perform operation
 * 0709: Action not permitted for missing KYC data
+* 0711: Verification completed. No missing requirements to fill
```
Comment (thenewversion) [inline source/index.html.md:221]: is 0710 used for something else? Can't see it in the PHP codebase

Comment (viccomx) [reply source/index.html.md:221]: Yes, https://github.com/bitsoex/bitsoex/pull/3822 will be used here, whenever I can merge those PRs

Comment (domix) [reply source/index.html.md:222]: I saw @viccomx added the 0710 [here](https://github.com/bitsoex/slate/commit/1964a30fd137658da88367befd0d75038dd4baec) @thenewversion

Comment (thenewversion) [reply source/index.html.md:222]: Of course

**Pull request merged**